specification/cognitiveservices/ContentSafety/tspconfig.yaml

```diff
@@ -26,6 +26,6 @@ options:
   #   packageDetails:
   #     name: "@azure-rest/contentsafety-rest"
   #     description: "ContentSafety Service"
-  # "@azure-tools/typespec-java":
-  #   package-dir: "contentsafety"
-  #   namespace: com.contentsafety
+  "@azure-tools/typespec-java":
+    package-dir: "azure-ai-contentsafety"
+    namespace: com.azure.ai.contentsafety
```

Comment:
BTW, you can add `examples-directory: examples` here (same as typespec-autorest config) to let emitter generate some sample/test for your reference. (you likely still need to improve them, like your doc, but it should give you an idea of how SDK works)